internal/app/app.go

```diff
@@ -18,22 +19,36 @@ func New() (*App, error) {
 	var err error
 	a := &App{}
 
+	// Setup logger
```

Comment:
I steal the logger idea from `clean template` [I don't know which is the idiomatic way to configure logger [especially for 3th party libraries]]